tests/test_downloaders.py

```diff
@@ -384,6 +385,19 @@ def test_the_correct_call_of_check_and_download_url_without_file_prefix(self):
             download_manager.download_dataset(dataset, 'testing_value')
             self.assertIsNone(mock_dl_url.call_args[1]['file_prefix'])
 
+    def test_the_storing_ability_of_file_local_address(self):
+        """
+        Test that address of downloaded file is added to the dataseturi model.
+        """
+        download_manager = downloaders.DownloadManager(save_path=True)
+        dataset = Dataset.objects.get(pk=3)
+        with mock.patch.object(downloaders.HTTPDownloader, 'check_and_download_url') as mock_dl_url:
+            mock_dl_url.return_value = ('test.nc', True)
+            download_manager.download_dataset(dataset, 'testing_value')
+            self.assertEqual(dataset.dataseturi_set.filter(
+                                dataset=dataset,service=LOCAL_FILE_SERVICE)[0].uri,
+                            'testing_value/test.nc')
```

Comment:
As stated in another comment, the stored path should be absolute.

Reply:
Done in the next commit